server/sonar-server/src/main/java/org/sonar/server/issue/workflow/IssueWorkflow.java

```diff
@@ -177,9 +180,7 @@ public boolean doTransition(DefaultIssue issue, String transitionKey, IssueChang
 
   public List<Transition> outTransitions(Issue issue) {
     State state = machine.state(issue.status());
-    if (state == null) {
-      throw new IllegalArgumentException("Unknown status: " + issue.status());
-    }
+    checkArgument(state != null, "Unknown status: %s", issue.status());
     return state.outManualTransitions(issue);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) NullPointerException might be thrown as 'state' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2259)